src/System.Linq/src/System/Linq/DebugView.cs

```diff
@@ -12,7 +11,10 @@ namespace System.Linq
     // NOTE: DO NOT DELETE THE FOLLOWING DEBUG VIEW TYPES.
     // Although it might be tempting due to them not be referenced anywhere in this library,
     // Visual Studio currently depends on their existence to enable the "Results" view in 
-    // watch windows.
```

Comment:
This could be generalised as "IDE's depend". I'm pretty sure SharpDevelop, Xamarin Studio and MonoDevelop all use them too.